main.js

```diff
@@ -1,9 +1,88 @@
+// Creating constants
+
+const openModal = document.getElementById('openDesktop');
+const closeModal = document.getElementById('closeDesktop');
 const open = document.querySelector('.menu__icon');
+const close = document.querySelector('.cls');
+
+// Menu Project
+
 open.addEventListener('click', () => {
```

Comment:
- [ ] Only the first popup for the first project opens when clicking on the ```See project``` button, kindly implement this function for the rest of the projects